src/main/java/org/mockito/Mockito.java

```diff
@@ -2702,4 +2696,17 @@ static MockitoDebugger debug() {
     public static MockitoFramework framework() {
         return new DefaultMockitoFramework();
     }
+
+    /**
+     * Allows to manage Mockito strictness (debugging, cleaner tests) without JUnit.
+     * <p>
+     * TODO javadoc
+     *
+     * @param testClassInstance
+     * @param strictness
+     */
+    @Incubating
+    public static MockitoMocking startMocking(Object testClassInstance, Strictness strictness) {
+        return new DefaultMockitoMocking(testClassInstance, strictness, new ConsoleMockitoLogger());
+    }
```

Comment:
I like a lot the idea of a builder. "MockitoMocking" name is somewhat meaningless. How about we call it "MockitoSession" or "MockitoRun" to indicate a lifecycle of some kind with clear starting and ending point:

```java
MockitoSession mockito = Mockito.mockitoSession().initMocks(this).strictness(Strictness.STRICT_STUBS).startMocking();
mockito.finishMocking();
```

I'm leaning towards new method on Mockito class instead for new static class endpoint. This way, the API is easier to discover / easier to use.